src/config/test_postake_txns.mlh

```diff
@@ -0,0 +1,17 @@
+[%%import "config/ledger_depth/full.mlh"]
+[%%import "config/curve_size/small.mlh"]
+[%%import "config/coinbase/standard.mlh"]
+[%%import "config/consensus/postake_tiny.mlh"]
+[%%import "config/scan_state/small.mlh"]
+[%%import "config/debug_level/some.mlh"]
+[%%import "config/proof_level/check.mlh"]
```

Comment:
what did you decide with @bkase about the proof level for this? we could turn down the ledger depth a bit to help some with slot times.